### PR TITLE
Login: Move everything into a Login folder, refactor

### PR DESCRIPTION
**Pull request by illume**
Work in progress, please don't merge.

- [x] Add a Login.stories.tsx
- [x] Separate components for presentation and logic
- [x] Move the login-image into the Login folder
- [x] Move from html tags and css to mui components

How to test:
- `npm run storybook`
- have a look at the web app link in the PR

### CODE REVIEW

File: FrontEnd/src/components/Login/Login.tsx
Comment (esscott) [inline FrontEnd/src/components/Login/Login.tsx]: Only thing is, would this make more sense as a button? Otherwise, this all looks great and is a clear example of the steps that need to be taken in order to refactor the rest of the components

**Review by esscott — CHANGES_REQUESTED** [review]
Please see my question in Login.tsx about changing link to MUI button. Once that's addressed, this is ready to merge after Alex's presentation today! Thanks for leading the refactor train René 😺

File: FrontEnd/src/components/Login/Login.tsx
Comment (illume) [reply FrontEnd/src/components/Login/Login.tsx]: The way I thought it should be a link rather than a button is: it navigates to a /login URL rather than doing some sort of action. But also... it's the main call to action on the view... so maybe a button would be more appropriate. hrmm.

**Review by esscott — APPROVED** [review]

File: FrontEnd/src/components/Login/Login.tsx
Comment (illume) [reply FrontEnd/src/components/Login/Login.tsx]: We decided to go with the Button.

**Pull request merged**
